modules/data_team_helper/jsx/behavioural_qc_module.js

```diff
@@ -143,6 +143,32 @@ var IncompleteCandidatesRow = React.createClass({
 });
 
 var InstrumentConflictsRow = React.createClass({
+    handleClick: function handleClick(event) {
+        event.preventDefault();
+        var row = this.props.row;
+        var candID= row.CandID;
+        var instrument=row.TableName;
+        var question = row.FieldName;
+        var form = $('<form />', {
```

Comment:
any reason in particular why you're not using `loris.loadFilteredMenuClickHandler` here?